src/ProjectOrigin.WalletSystem.Server/Models/CertificateViewModel.cs

```diff
@@ -2,6 +2,7 @@
 using System.Collections.Generic;
 using System.Linq;
 using Google.Protobuf.WellKnownTypes;
+using NBitcoin.RPC;
```

Comment:
why is this included now?

Reply:
Think you are looking at an outdated version